components/link/index.css

```diff
@@ -45,11 +46,14 @@ governing permissions and limitations under the License.
   }
 }
 
-.spectrum-Link--subtle,
-/** @deprecated */.spectrum-Link--quiet {
+.spectrum-Link--secondary {
   text-decoration: underline;
```

Comment:
I think this is duplicative? We already have `text-decoration: underline` above

Reply:
I've cleaned up the duplicated `text-decoration` property.